cli/add.go

```diff
@@ -162,8 +161,8 @@ func validateEnv(key, value string, envList []models.EnvironmentItemModel) (stri
 	valueSizeInBytes := len([]byte(value))
 	if configs.EnvBytesLimitInKB > 0 {
 		if valueSizeInBytes > configs.EnvBytesLimitInKB*1024 {
-			valueSizeInKB := ((float64)(valueSizeInBytes)) / 1024.0
-			return fmt.Sprintf("environment var (%s) value is too large (%#v KB), max allowed size: %#v KB", key, valueSizeInKB, (float64)(configs.EnvBytesLimitInKB)), nil
```

Comment:
This is an interesting change, I sense, that we did error handling wrong here. Instead of returning an error for too large env var values, we returned the message as the validated env var value and no error.